src/content/configuration/output.md

```diff
@@ -154,6 +156,15 @@ devtoolModuleFilenameTemplate: info => {
 If multiple modules would result in the same name, [`output.devtoolFallbackModuleFilenameTemplate`](#output-devtoolfallbackmodulefilenametemplate) is used instead for these modules.
 
 
+## `output.devtoolNamespace`
+
+`string`
+
+This option determines the modules namespace used with the [`output.devtoolModuleFilenameTemplate`](#output-devtoolmodulefilenametemplate). When not specified, it will default to the value of: [`output.library`](#output-library). It's use is to prevent source file path collisions in sourcemaps when loading multiple libraries built with webpack.
```

Comment:
@EugeneHlushko Just a small thing to make it more readable.

> It's used to prevent source file path collisions in sourcemaps when loading multiple libraries built with webpack.